playbooks/tasks/portals-prepare.yml

```diff
@@ -26,4 +25,12 @@
     state: directory
     owner: "user"
     group: "user"
-    mode: '0700'
\ No newline at end of file
+    mode: "0700"
+
+# Install rsync for synchronize commands
+- name: "Installing Rsync"
```

Comment:
@firyx can you help me with this? the command is successful but I get a path error when the `synchronize` command runs
```
Failed to find required executable rsync in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

I believe the issue is that rsync is not being installed properly in the docker container with this command, because rsync is installed locally and on the server and I can rsync a file to the server manually just fine.